prime-router/metadata/schemas/iPatientCare/iPatientCare-covid-19.schema

```diff
@@ -16,6 +16,8 @@ elements:
     - name: processing_mode_code
       csvFields: [{name: processing_mode_code}]
 
+    - name: message_id
+      csvFields: [{name: Accession_no}]
```

Comment:
Remove this line and just let the Reddy schema assign the csvField

Reply:
Done.